README.md

```diff
@@ -80,15 +80,15 @@ Public/index.html handles the UI layout for the web page, and references the nec
         ```json
         "env": {
             "NODE_ENV": "local",
-            "IotHubConnectionString": "<your iot hub connection string>",
+            "IotHubConnectionString": "<your IoT hub's connection string>",
             "EventHubConsumerGroup": "<your consumer group name>"
         }
         ```
 
 1. In the same directory as package.json, run `npm install` to download and install referenced packages.
 
 1. Run the website one of the following ways:
-    - From the command-line use `npm start`
+    - From the command-line with environment variables set, use `npm start`

```

Comment:
nit: I think there should be a comma  between "command-line" and "with". Up to you tho.

Reply:
I'll wrap it in parenthesis, since it is a bit of an aside to the user